entrypoint.sh

```diff
@@ -78,6 +85,9 @@ api_request() {
     log "$errors"
     cat "$err" 1>&2
     return 1
+  else
+    # Linear back-off for Shopify API requests
+    sleep $sleep_time
```

Comment:
I see this but there's also no loop. Shouldn't there be a loop if you're trying to do a backoff? A retry?